mmdet/core/fp16/deprecated_fp16_utils.py

```diff
@@ -0,0 +1,47 @@
+import warnings
+
+from mmcv.runner import (Fp16OptimizerHook, auto_fp16, force_fp32,
+                         wrap_fp16_model)
+
+
+class Depr_Fp16OptimizerHook(Fp16OptimizerHook):
+    """A wrapper class for the FP16 optimizer hook. This class wraps
+    Fp16OptimizerHook in "mmcv.runner" and shows a warning that the
```

Comment:
:class:`Fp16OptimizerHook`,  ``mmcv.runner``, ``mmdet.core`` might be better for docstring syntax.